lesson4/conftest.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+import requests
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--url",
+        action="store",
+        default="https://jsonplaceholder.typicode.com/todos",
+        help="Enter url"
+    )
+
+
+@pytest.fixture(scope="module")
+def base_url(request):
+    return request.config.getoption("--url")
+
+
+@pytest.fixture(scope="module")
+def session():
+    return requests.Session()
+
+
+@pytest.fixture(params=[1, 100, 200])
+def fixture_id_resource_positive(request):
+    return request.param
+
+
+@pytest.fixture(params=[-1, 0, 201, "string", False, [1, 2, 3], None])
+def fixture_id_resource_negative(request):
+    return request.param
+
+
+@pytest.fixture(params=[{"title": "A", "completed": True, "userId": 1},
+                        {"title": "long_title_long_title_long_title_long_title_long", "completed": False, "userId": 10},
+                        {"title": "Medium title", "completed": True, "userId": 10},
+                        {"title": " Title   with   wihtspaces ", "completed": False, "userId": 11},
+                        {"title": "Title with symbols *^[]/;< ", "completed": True, "userId": 5}])
+def fixture_payload_resource_positive(request):
+    return request.param
+
+
+@pytest.fixture(params=[1, 5, 10])
+def fixture_test_filtering_positive_by_userid(request):
```

Comment:
I think it's better to use a shorter name for this fixture, for instance, "userid_positive"